content/html/concepts/semantic-html/terms/details/details.md

```diff
@@ -0,0 +1,44 @@
+---
+Title: '<details>'
+Description: 'Encapsulated the <summary> element and any additional details which are only visible when the element is in an open state.'
+Subjects:
+  - 'Web Development'
+  - 'Web Design'
+Tags:
+  - 'Semantic'
+  - 'Tags'
+CatalogContent:
+  - 'learn-html'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The `<details>` semantic HTML element encapsulated the <summary> element and any additional elements which are only visible when the element is in an open state.
+There is a small triangle which shows the state of the element and by clicking on the element or using the space bar you can toggle between revealing and hiding additional information.
+
+## Syntax
+
+The `<details>` element can be in 2 states, open and closed. When in the closed state, only the content in the `<summary>` element is visible.
```

Comment:
Can we provide a sentence here that states that whenever we want to use the `<details>` element successfully, we _must_ provide a `<summary>` element?